demo/talkingdata-adtracking-fraud-detection/train_and_serve.py

```diff
@@ -1,13 +1,14 @@
+"""Module of training and request to predict server"""
 import gc
 import os
 import time
 import glob
 import requests
 # fmt:off
-import openmldb
```

Comment:
don't delete `import openmldb`. It may get seg fault in some machine

Reply:
Done.